hello/views.py

```diff
@@ -24,7 +24,7 @@ def category(request):
         'hello/category.html',
         {
             'name': SpotipyFunctions.displayname,
-            'followers': SpotipyFunctions.followers,
+            'picture': SpotipyFunctions.profile_pic,
```

Comment:
Need to catch the exception if the user does not have a spotify profile picture. Have a default photo for those users?